src/components/CreateCourse/CreateCourse.jsx

```diff
@@ -66,8 +72,7 @@ const CreateCourse = ({
 			name: author.name,
 		};
 		if (newAuthor.name.length > 2) {
-			setAuthorsList((prevAuthorsList) => [...prevAuthorsList, newAuthor]);
-			setAuthors((prev) => [...prev, newAuthor]);
+			dispatch(addAuthorAction([...authors, newAuthor]));
```

Comment:
it should be done in the reducer, not here 

```suggestion
			dispatch(addAuthorAction(newAuthor));
```